src/todos.js

```diff
@@ -0,0 +1,153 @@
+
+    const deleteButton = document.createElement('button');
+    deleteButton.id = 'delete-button';
+    const deleteIcon = document.createElement('i');
+    deleteIcon.classList.add('fas', 'fa-trash');
+    deleteButton.appendChild(deleteIcon);
+    todoItemContainer.appendChild(deleteButton);
+
+    deleteButton.addEventListener('click', () => {
+      const parent = deleteButton.parentElement;
+      parent.parentElement.removeChild(parent);
+      removeHorizontalLine(todoItemId);
+
+      todos = todos.filter((item) => item.index !== todoItem.index);
+
+      todos.forEach((item, index) => {
+        item.index = index + 1;
+        const itemId = `todo-item-${item.index}`;
+        const todoItemContainer = document.getElementById(itemId);
+        if (todoItemContainer) {
+          const todoText = todoItemContainer.querySelector('#todo-text');
+          const hrId = `${itemId}-hr`;
+          const horizontalLine = document.getElementById(hrId);
+          todoItemContainer.id = `todo-item-${item.index}`;
+          todoText.id = 'todo-text';
+          horizontalLine.id = `${itemId}-hr`;
+        }
+      });
+
+      saveTodosToLocalStorage();
+    });
+
+    checkbox.addEventListener('change', () => {
+      todoItem.completed = checkbox.checked;
+      saveTodosToLocalStorage();
+
+      todos.forEach((item, index) => {
+        item.index = index + 1;
+        const itemId = `todo-item-${item.index}`;
+        const todoItemContainer = document.getElementById(itemId);
+        if (todoItemContainer) {
+          const todoText = todoItemContainer.querySelector('#todo-text');
+          const hrId = `${itemId}-hr`;
+          const horizontalLine = document.getElementById(hrId);
+          todoItemContainer.id = `todo-item-${item.index}`;
+          todoText.id = 'todo-text';
+          horizontalLine.id = `${itemId}-hr`;
+        }
+      });
+    });
+
+    const hr = document.createElement('hr');
+    hr.id = `${todoItemId}-hr`;
+    todosContainer.appendChild(hr);
+  }
+
+  function removeHorizontalLine(todoItemId) {
+    const hrId = `${todoItemId}-hr`;
+    const horizontalLine = document.getElementById(hrId);
+    if (horizontalLine) {
+      horizontalLine.parentElement.removeChild(horizontalLine);
+    }
+  }
+
+  const clearButton = document.querySelector('.clearer');
+
+  clearButton.addEventListener('click', () => {
+    todos = todos.filter((item) => !item.completed);
+
+    todosContainer.innerHTML = '';
+    todos.forEach((todoItem, index) => {
+      todoItem.index = index + 1;
+      renderTodoItem(todoItem);
+    });
+
+    saveTodosToLocalStorage();
+  });
+
+  function saveTodosToLocalStorage() {
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+
+  function refreshIt() {
+    const refreshIcon = document.querySelector('.refresh-it');
+    refreshIcon.addEventListener('click', () => {
+      todos = [];
+      todosContainer.innerHTML = '';
+      localStorage.removeItem('todos');
+    });
+  }
```

Comment:
- [Optional] I noticed that when I click on the refresh button it removes all the items from the list, the refresh button is supposed to refresh the page and not delete the items. Please kindly fix that 🙏🏾